src/datasets/utils/resources/languages.json

```
@@ -682,6 +682,7 @@
     "jv-ID": "Javanese (Indonesia)",
     "jv-Java": "Javanese (Javanese)",
     "jv-x-bms": "Banyumasan",
+    "jya": "Japhug",
```

Please note the official name corresponding to the tag "jya" is "Jiarong": Japhug is one of the Jiarong languages: 
- https://iso639-3.sil.org/code/jya
- https://en.wikipedia.org/wiki/Gyalrong_languages#Languages

```suggestion
    "jya": "Jiarong",
```

---

After a discussion with the linguistic experts, we decided to use the Glottolog codes, which are more precise and therefore more robust because they avoid potential future ambiguities...

---

If I may 'jump in': a more general topic is, Which way to go for identifying language varities. It's a big topic. Glottolog and Ethnologue both have information on language groupings. Glottolog is more fine-grained and the database maintainers are more transparent about how decisions are made, so, strange as the codes may seem ('japh1234' looks more unwieldy than 'jya'), they look like a good way to go. An issue was created [here](https://github.com/huggingface/datasets/issues/4881) in the hope of bringing attention to this topic.

---

And I just added the ISO 639-3 codes too since they are requested in the dataset card.

---

Yes, I agree the issue page #4881 is the right approach to propose this change to discuss and agree on it before incorporating to the library. :)

---

@BenjaminGalliot are you working on a specific dataset addition?

---

Yes, called [pangloss](https://huggingface.co/datasets/Lacito/pangloss)

---

Great dataset, @BenjaminGalliot! 

Please note that our `languages.json` list is used to validate only legacy datasets present on this GitHub repo.

Currently, datasets (as your case) are added to the Hub instead, and their validation is done by the Hub team: our `languages.json` file is not used by them, although we try to be on sync.
- I have pinged that team on the issue discussion page: #4881

In the meantime (the change of language codes could take some time), I would recommend you to used the `language_details` tag instead, something like:
```
language_details: 
```
language_details: "'jya' consists of regular 'japh1234'"
```

---

OK, I removed the `language_bcp47` field and added the `language_details` one!